apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,292 @@
+   * events.
+   */
+  public static final String PATIENT_PARTD_CONTRACT_VALUE =
+      """
+    When searching for a Patient's Part D events information, this resource identifies
+    the Part D contract value that will be used when determining eligibility.
+    Example:
+       - `_has:Coverage.extension=ABCD`""";
+
+  /**
+   * Open API short description for /Patient's Part D reference year to be used in determining Part
+   * D events.
+   */
+  public static final String PATIENT_PARTD_REFYR_SHORT = "Part D contract reference year";
+  /**
+   * Open API content value for /Patient's Part D reference year to be used in determining Part D
+   * events.
+   */
+  public static final String PATIENT_PARTD_REFYR_VALUE =
+      """
+    When searching for a Patient's Part D events information, this resource identifies
+    the reference year that will be applied when determining applicable Part D events.
+    Example:
+       - `_has:Coverage.rfrncyr=2023`""";
+
+  /**
+   * Open API short description for /Patient's Part D event data; provides a URI for the _next_ set
+   * of data.
+   */
+  public static final String PATIENT_PARTD_CURSOR_SHORT = "The cursor used for result pagination";
+  /**
+   * Open API content value for /Patient's Part D event data; provides a URI for the _next_ set of
+   * data.
+   */
+  public static final String PATIENT_PARTD_CURSOR_VALUE =
+      """
+     Provide a pagination cursor for processing Patient's Part D events information; this resource identifies
+     a numeric offset into a result set.
+     Example:
+        - `cursor=200`""";
+
+  /** Open API content short description for /Patient's identifier parameter. */
+  public static final String BENEFICIARY_SP_RES_ID_SHORT =
+      "Identifier resource for the covered party";
+  /** Open API content value for /Patient's identifier parameter. */
+  public static final String BENEFICIARY_SP_RES_ID_VALUE =
+      """
+    Fetch _Beneficiary_ data using a FHIR _IdType_ identifier; an IdType
+    represents the logical identity for a resource, or as much of that.
+    identity that is known.
+
+    In FHIR, every resource must have a _logical ID_ which is defined by the FHIR specification as:
+    `<code>Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),
+    '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID
+    or any other identifier pattern that meets these constraints.)</code>`
+
+    This class contains that logical ID, and can optionally also contain a relative or absolute URL
+    representing the resource identity; the following are all valid values for IdType, and all might
+    represent the same resource
+
+    Examples:
+      - `<code>123</code> (just a resource's ID)`
+      - `<code>beneficiary/123</code> (a relative identity)`
+      - `<code>http://example.com/beneficiary/123 (an absolute identity)</code>`""";
```

Comment:
I think we should probably forego the separate BENEFICIARY_SP definitions here and just re-use the PATIENT_SP definitions.